compiler-rt/lib/msan/msan.cpp

```diff
@@ -352,10 +352,44 @@ void __sanitizer::BufferedStackTrace::UnwindImpl(
 
 using namespace __msan;
 
+#define PRINT_FAULTING_INSTRUCTION(instname)                            \
+  if (__msan::flags()->print_faulting_instruction) {                    \
+    Printf("Instruction that failed the shadow check: %s\n", instname); \
+    Printf("\n");                                                       \
+  }
+
+#define WARN_IF_PRINT_FAULTING_INSTRUCTION_REQUESTED                    \
```

Comment:
Why can't this be a real function? This won't show up in the stack trace?

Reply:
Good point, fixed in https://github.com/llvm/llvm-project/pull/136539/commits/56791cfa8ec7c4d739d3dd34a861a7770437180b